File: 8KYU/Multiple of Index/solution.cs
using System;
using System.Collections.Generic;
public static class Kata
{
	public static List<int> MultipleOfIndex(List<int> xs)
	{
		var result = new List<int>();
		for (var i = 1; i < xs.Count; i++)
			if (xs[i] % i == 0)
				result.Add(xs[i]);

		return result;
	}
}

File: 8KYU/Multiple of Index/solution.go
package kata

func multipleOfIndex(ints []int) []int {
	var res []int
	for i := 1; i < len(ints); i++ {
		if ints[i]%i == 0 {
			res = append(res, ints[i])
		}
	}
	return res
}
